packages/shared/src/components/modals/common/Modal.tsx

```diff
@@ -44,35 +47,33 @@ const modalKindAndSizeToOverlayClassName: Partial<
   Record<ModalKind, Partial<Record<ModalSize, string>>>
 > = {
   [ModalKind.FlexibleTop]: {
-    [ModalSize.Small]: 'pt-10 mobileL:pt-20',
-    [ModalSize.Medium]: 'pt-10 mobileL:pt-20',
+    [ModalSize.Small]: 'tablet:pt-20',
+    [ModalSize.Medium]: 'tablet:pt-20',
```

Comment:
Are these just generic modal changes in regards to mobileUX?

Reply:
Yes, all of our Modal is going to be either full screen or drawer on mobile. No more specific width sizes.